smartcontract/service/native/ontid/ontid_test.go

```diff
@@ -486,3 +496,167 @@ func CaseGetDocument(t *testing.T, n *native.NativeService) {
 	}
 	fmt.Println(string(res))
 }
+
+func CaseBeforeNewApiFork(t *testing.T, n *native.NativeService) {
+	// 1. register ID
+	id0, _ := account.GenerateID()
+	a0 := account.NewAccount("")
+	id1, _ := account.GenerateID()
+	a1 := account.NewAccount("")
+	id2, _ := account.GenerateID()
+	a2 := account.NewAccount("")
+	if err := regID(n, id0, a0); err != nil {
+		t.Fatal("register ID error", err)
+	}
+	if err := regID(n, id1, a1); err != nil {
+		t.Fatal("register ID error", err)
+	}
+	if err := regID(n, id2, a2); err != nil {
+		t.Fatal("register ID error", err)
+	}
+
+	// 2. add context before fork height
+	var contexts = [][]byte{[]byte("https://www.w3.org/ns0/did/v1"), []byte("https://ontid.ont.io0/did/v1"), []byte("https://ontid.ont.io0/did/v1")}
+	context := &Context{
+		OntId:    []byte(id0),
+		Contexts: contexts,
+		Index:    1,
+	}
+	sink := common.NewZeroCopySink(nil)
+	context.Serialization(sink)
+	n.Tx.SignedAddr = []common.Address{a0.Address}
+	_, err := n.NativeCall(utils.OntIDContractAddress, "addContext", sink.Bytes())
+	assert.ErrorContains(t, err, "doesn't support this function")
```

Comment:
use assert from `testify/assert`: `assert.Contains(t, err.Error(), "doesn't support this function")`